protocol-units/bridge/chains/ethereum/src/client.rs

```diff
@@ -472,3 +473,46 @@ impl BridgeContractCounterparty for EthClient {
 		}))
 	}
 }
+
+fn test_wrapping_to(a: &U256, b: u64) {
+    assert_eq!(a.wrapping_to::<u64>(), b);
+}
```

Comment:
We don't need this as a util fn outside of `mod tests {}` you can pop it inside the test module.

Reply:
Does it break the code if it's outside? why not expose it so that you could use it when building some tests by yourself?

Reply:
We can do that but we would want to feature flag it. Any code for testing shouldn't be built in the release binary, because it adds bloat and could open up security attack vector.

Reply:
But no it doesn't break anything.